Adds Glimmer 2 Betas (#594)
ember-learn/ember-blog: source/2020-04-03-the-ember-times-issue-142.md

---

## [Betas of Glimmer.js 2 🎉](https://github.com/glimmerjs/glimmer.js/releases)

A set of **new betas for Glimmer.js 2.0.0** ([beta.1](https://github.com/glimmerjs/glimmer.js/releases/tag/v2.0.0-beta.1) && [beta.3](https://github.com/glimmerjs/glimmer.js/releases/tag/v2.0.0-beta.3)) have been released! 🔥🔥🔥 

> The releases incorporate some exciting new developments including a new minimal `renderComponent` API, a template import based design, a standardized build based on plain JavaScript modules, a new Glimmer.js app blueprint and support for custom helpers and modifiers. The standardized build unlocks the ability to use whatever custom build pipeline you might want (WebPack, Parcel, Rollup, etc.)!

Reviewer: Can we mention in the standardized build section that this emulates the goals of the Embroider work for Ember? Not sure if that **exactly** fits here, but it seems important to draw parallels to the “same feature” in the Ember side.

> The releases incorporate some exciting new developments including a new minimal `renderComponent` API, a template import based design, a standardized build based on plain JavaScript modules, a new Glimmer.js app blueprint and support for custom helpers and modifiers. The standardized build unlocks the ability to use whatever custom build pipeline you might want (WebPack, Parcel, Rollup, etc.)!

Reviewer 2: Because there are several examples of new developments, I think adding a comma before the last example will help with reading.
```suggestion
The releases incorporate some exciting new developments including a new minimal `renderComponent` API, a template import based design, a standardized build based on plain JavaScript modules, a new Glimmer.js app blueprint, and support for custom helpers and modifiers. The standardized build unlocks the ability to use whatever custom build pipeline you might want (WebPack, Parcel, Rollup, etc.)!
```

> The Glimmer.js betas could pave the way for what we might want to do in Ember.js, or even what we might _not_ want to do in Ember.js. With that in mind, please note that these new Glimmer.js betas are definitely experimental 🔬🧪 and should while have fun exploring them 🤠, but also keep that in mind when considering them for use in your application.

Reviewer 2: I wasn't sure what you meant by "and should while", so I omitted it in the suggestion below. Can you check to see if it matches your intent?
```suggestion
The Glimmer.js betas could pave the way for what we might want to do in Ember.js, and even what we might _not_ want to do in Ember.js. Please note that these new Glimmer.js betas are definitely experimental 🔬🧪Have fun exploring them 🤠, but keep this in mind when considering them for use in your application.
```
